src/makeCalculator.js

```diff
@@ -36,8 +36,52 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
-}
+  let result = 0;
+
+  return {
+    add(number) {
+      result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      result -= number;
+
+      return this;
+    },
+
+    multiply(number) {
+      result *= number;
+
+      return this;
+    },
 
+    divide(number) {
+      if (number !== 0) {
+        result /= number;
+      }
+
+      return this;
+    },
+
+    reset() {
+      result = 0;
+
+      return this;
```

Comment:
I think that only operate method needs to return this, you only pass those methods into operate